Problem2/src/problem2/Problem2.java

```diff
@@ -10,12 +10,39 @@
  * @author cgallinaro
  */
 public class Problem2 {
-
+    
     /**
-     * @param args the command line arguments
+     * determines weather the integer passed in is divisible by three by recursion
+     * sums up all of the digits, if they total to 3, 6, or 9, returns true,
+     * else false if the number is less than ten, calls self until base cases met
+     * @param d the number to check
+     * @return true if divisible, else false
      */
-    public static void main(String[] args) {
-        // TODO code application logic here
+    public static boolean isDivisibleByThree(int d) {
+        //check the base cases of 3, 6, and 9
+        if (d == 3 ||3 == 6 || 3 == 9) {
```

Comment:
You if statement should read "d == 3 || d == 6 || d == 9".  The tests you wrote fail because of this typo.